core/src/test/java/com/graphhopper/routing/AlternativeRouteEdgeCHTest.java

```diff
@@ -141,9 +141,10 @@ public void testCalcOtherAlternatives() {
         hints.putObject("alternative_route.max_paths", 4);
         AlternativeRouteEdgeCH altDijkstra = new AlternativeRouteEdgeCH(g.getRoutingCHGraph(), hints);
         List<AlternativeRouteEdgeCH.AlternativeInfo> pathInfos = altDijkstra.calcAlternatives(10, 5);
-        assertEquals(2, pathInfos.size());
+        assertEquals(3, pathInfos.size());
         assertEquals(IntArrayList.from(10, 4, 3, 6, 5), pathInfos.get(0).path.calcNodes());
         assertEquals(IntArrayList.from(10, 4, 8, 7, 6, 5), pathInfos.get(1).path.calcNodes());
+        assertEquals(IntArrayList.from(10, 4, 8, 7, 6, 3, 2, 9, 1, 5), pathInfos.get(2).path.calcNodes());
```

Comment:
@michaz  What do you think about this? The third alternative does not make so much sense because its just like the second but with an extra little detour. This probably now occurs because the shortcuts are slightly different. Or do you think I might have broken something? Defining what an 'acceptable' alternative is on such a small graph is probably rather hard and/or the heuristics aren't even meant to work for such small graphs?

Reply:
Yeah, that‘s probably okay. Can‘t think of something you could have broken here.